transformers/events/vat_grab/converter_test.go

```diff
@@ -29,10 +31,11 @@ import (
 )
 
 var _ = Describe("Vat grab converter", func() {
-	var converter vat_grab.VatGrabConverter
+	var converter vat_grab.Converter
+	db := test_config.NewTestDB(test_config.NewTestNode())
```

Comment:
Liked Gabe's comment on another PR - the `BeforeEach` should clean the test db. Also, I think it'd be good to explicitly declare the `db` a `var`. Could be wrong, but I feel like I've run into scoping issues when something like this is declared implicitly